pkg/kubelet/volumemanager/volume_manager.go

```diff
@@ -380,6 +382,27 @@ func (vm *volumeManager) WaitForAttachAndMount(pod *v1.Pod) error {
 	return nil
 }
 
+// Figure out the volume's unique name to find out whether it's still being set-up
+func (vm *volumeManager) isVolumeBeingSetUp(outerVolumeSpecName string, volumesToMount []cache.VolumeToMount) bool {
+	for _, volumeToMount := range volumesToMount {
```

Comment:
instead of looping through all "volumesToMount" for each unmountedVolume, you could loop less and just create a set/list of volumes being set up, right now I think there's redundant looping

Reply:
The looping happens only in the case the volume mount times-out. This is not a standard situation, I'm not sure keeping another list in the memory to speed up spitting an error message is worth it.

Reply:
OK. I don't have strong preference here so I will implement it as you suggest.